package.json

```diff
@@ -10,30 +10,52 @@
   "license": "MIT",
   "author": "Sanity.io <hello@sanity.io>",
   "sideEffects": false,
+  "type": "commonjs",
   "exports": {
     ".": {
-      "import": {
-        "types": "./dist/1.esm.d.ts",
-        "default": "./dist/1.esm.mjs"
+      "types": "./dist/index.d.ts",
+      "source": "./src/1.ts",
+      "require": "./dist/index.js",
+      "node": {
+        "import": "./dist/index.cjs.mjs",
+        "require": "./dist/index.js"
       },
-      "require": {
-        "types": "./dist/1.umd.d.ts",
-        "default": "./dist/1.umd.cjs"
+      "import": "./dist/index.mjs",
+      "default": "./dist/index.js"
+    },
+    "./1": {
```

Comment:
This lets consumers pin their `groq-js` to a specific GROQ version. While the main `.` export can just always point to the latest spec that's implemented.
Also worth noting if an app ends up doing both `import 'groq-js'` and `import 'groq-js/1'` our `@sanity/pkg-utils` tooling is smart enough to dedupe code between the export points and you don't end up doubling the memory usage and amount of code that's loaded up.

In fact, `dist/index.mjs` currently looks like this, since both exports use the same exact entrypoint:
```js
export { evaluate, parse } from './1.mjs';
```